src/library/commissioner_impl.cpp

```diff
@@ -1510,7 +1510,8 @@ Error CommissionerImpl::DecodeActiveOperationalDataset(ActiveOperationalDataset
         dataset.mPresentFlags |= ActiveOperationalDataset::kSecurityPolicyBit;
     }
 
-    aDataset = dataset;
+    dataset.mRawTlvs = aResponse.GetPayload();
```

Comment:
can we be sure that there is no other TLV in the response?

Reply:
By Thread spec,  there is no other TLVs in the response.